app/helpers/application_helper.rb

```diff
@@ -385,6 +385,12 @@ def show_parent?(obj)
     (obj.new_record? && obj.class.count > 0) || (!obj.new_record? && obj.class.count > 1)
   end
 
+  def documentation_button(section)
+    link_to(icon_text('question-sign', _('Documentation'), :class => 'icon-white'),
+            "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html##{section}",
```

Comment:
Nothing technical.  The only reason I haven't done it is that people send updates for 1.8 (for example) and will forget to update the 1.9 docs too.. if I leave copying to later, then I'm more certain that we haven't lost any of the changes during the 1.8 timeframe.

Katello uses a branching model, which I prefer as it'd make it much easier as you could easily cherry pick between branches, but that has other overhead in maintenance (and doesn't appear to be universally popular).

Ideas very welcome!